### PR TITLE
Implemented: Change the flag --project-manifest to --manifest-path #1333

### PR DESCRIPTION
**Pull request by amartyadav**
1. **crates/release_plz/src/args/release.rs** _AND_ **crates/release_plz/src/args/update.rs**

- Commented out (line 20 release.rs, line 22 update.rs):
  
  `#[arg(long, value_parser = PathBufValueParser::new())]` for `project_manifest: Option<PathBuf>`
  to still keep the project_manifest flag as requested in issue #1333 "Keep the old flag for cli retro-compatibility".
- Added (line 23 - 27 in release.rs, line 25 - 29 in update.rs):

  `#[arg(long, value_parser = PathBufValueParser::new())]
      manifest_path: Option<PathBuf>`
  with the same '--help' message as the old flag.
- Changed (line 139 release.rs, line 102 update.rs) -> from `self.project_manifest` to `self.manifest_path`
- Added (line 205 release.rs, line 243 update.rs) -> to include `manifest_path: None,`

2. **website/docs/github/quickstart.md**
- Renamed (line 217, line 245) -> the Github Action input parameter from `project_manifest` to `manifest_path`

Additional Notes:
- All tests passing (including Docker tests)
- Code formatted
- Followed all steps in [Contributing.MD](https://github.com/MarcoIeni/release-plz/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: crates/release_plz/src/args/update.rs [hunk 1]
```diff
@@ -19,8 +19,14 @@ pub struct Update {
...
     /// If not provided, release-plz will use the Cargo.toml of the current directory.
     /// Both Cargo workspaces and single packages are supported.
-    #[arg(long, value_parser = PathBufValueParser::new())]
+    // #[arg(long, value_parser = PathBufValueParser::new())]
     project_manifest: Option<PathBuf>,
+
+    /// Path to the Cargo.toml of the project you want to update.
+    /// If not provided, release-plz will use the Cargo.toml of the current directory.
+    /// Both Cargo workspaces and single packages are supported.
+    #[arg(long, value_parser = PathBufValueParser::new())]
```
Comment (MarcoIeni) [inline crates/release_plz/src/args/update.rs:28]:
```suggestion
    #[arg(long, value_parser = PathBufValueParser::new(), alias = "project-manifest")]
```
With a clap [alias](https://docs.rs/clap/latest/clap/struct.Arg.html#method.alias), you can delete the `project_manifest` variable 👍

Comment (amartyadav) [reply crates/release_plz/src/args/update.rs:28]: Thanks for the suggestion! I will do that and submit it.

Comment (amartyadav) [reply crates/release_plz/src/args/update.rs]: Hi @MarcoIeni, I implemented your suggestion. It should reflect in the latest commits. 
All tests are passing. Code formatting done as per Contributing.MD

Please have a look when you can. Cheers! 😄

**Comment by amartyadav** [comment]
@MarcoIeni, would it be possible to view the PR (and merge it if it addresses the issue in question)? It would be my first major PR merge to an open source project. Thank you!

**Comment by MarcoIeni** [comment]
I'll do it today. Sorry if it took so long, it's a busy period for me. Hopefully coming to an end now 👍

**Comment by MarcoIeni** [comment]
merged, thanks!

**Pull request merged**